### ISSUE TITLE
Check Web Accessibility

### ISSUE THREAD

**Issue by ghutchis** [pittquantum/backend#6]
We need to check for web accessibility:

http://wave.webaim.org
http://achecker.ca/checker/
http://www.cynthiasays.com
http://fae20.cita.illinois.edu

Anecdotally, a visually impaired staff member said the font size was "small" but was able to increase it easily.

**Comment by RitwikGupta** [comment]
I'll have to look up the accessibility guidelines. Let's do this check once Dr. Lambrecht's dev server is up and running.

**Comment by JoshuaRogan** [comment]
I agree this isn't a primary concern right now. I also already increased the font from 14-16px.

**Comment by ghutchis** [comment]
We need to revisit this. Please take a look particularly at the first two links, which indicate some issues, most should be fixable.

**Comment by ghutchis** [comment]
http://wave.webaim.org/report#/pqr.pitt.edu

**Comment by RitwikGupta** [comment]
99% of the accessibility "errors" presented by these analyzers is just how Bootstrap uses tags in a broad manner. `a` tags may necessarily not be links, etc. I'm not sure how to go about this without making the site look ugly.

**Comment by JoshuaRogan** [comment]
I can make some changes to cut down on them. But you are right I don't
think we will be able to remove all of them.

On Sun, Jul 12, 2015 at 3:27 AM Ritwik Gupta <notifications@github.com>
wrote:

> 99% of the accessibility "errors" presented by these analyzers is just how
> Bootstrap uses tags in a broad manner. a tags may necessarily not be
> links, etc. I'm not sure how to go about this without making the site look
> ugly.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/pittquantum/PittQuantumRepository/issues/6#issuecomment-120698613>
> .
>
-- 
Josh Rogan
JoshuaRogan.com

**Comment by JoshuaRogan** [comment]
All of the empty links that are being flagged for not having text do have title text. From what I have been looking up this is acceptable for accessibility.

**Comment by blueocto** [comment]
Hi @JoshuaRogan the title attribute (as far as I've tested) isn't read aloud on screenreaders and is meant to be an accompaniment to actual text inside the <a> tags, not instead of.

The W3C mention "For each anchor element that has a title attribute, check that the title attribute together with the link text describes the purpose of the link."

Scott O'Hara wrote an amazingly thorough article on the complexities of the `title` attribute here: https://www.24a11y.com/2017/the-trials-and-tribulations-of-the-title-attribute/
